src/App.css

```diff
@@ -1,38 +1,68 @@
-.App {
-  text-align: center;
-}
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+
+.blue { /*blue divs*/
+  background-color: #1EBDF4;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  padding: 1em;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.grey { /*grey divs*/
+  background-color: #d3d3d3;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  padding: 1em;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.card {
   display: flex;
   flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  background-color: white;
+  border-radius: 7px;
+  width: 95%;
+  padding: 1em;
+  opacity: 0.95;
+  box-shadow: 0px 0px 10px 2px rgba(30, 30, 30, 0.2);
+  justify-content: flex-start;
+  
 }
 
-.App-link {
-  color: #61dafb;
+.row {
+  display: flex;
+  flex-direction: row;
+  flex-flow: row wrap;
+  justify-content: space-evenly;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+h1 {
+  font-size: 30px;
+  margin-top: 0;
+  color: #1c1a19;
 }
+
+h2 {
+  font-size: 20px;
+  color: #1c1a19;
+}
+
+p {
+  font-size: 18px;
+  line-height: 1.5;
+  color: #1c1a19;
+}
+
+em {
+  font-style: normal;
+  font-weight: bolder;
+}
+
+#flamingo {
+  width: 30%;
+}
```

Comment:
This is why the flamingo stretches kind of weirdly on smaller viewports (as you mentioned in the PR). I would personally recommend using CSS Grid (e.g. `grid-template-columns`) to split up the left and right sides of the card (in percentages, so something like `40% 60%`) - you can always add gutters or padding/margin for spacing - and then make the image itself (e.g. `#flamingo`) just always fill `100%` of the width of its container (with `height: auto`). This will make sure that the aspect ratio stays the same no matter how large the screen is!